src/main.js

```diff
@@ -95,6 +96,26 @@ function getNLSConfiguration() {
 	return { locale: initialLocale, availableLanguages: {} };
 }
 
+function migrateUserDataDir(newDir) {
+	var oldDir = path.join(process.env.HOME, '.config', product.nameShort);
+	if (!fs.existsSync(oldDir)) {
+		return;
```

Comment:
@Tyriar also keep in mind the note on 0.37.7 about setPath behavior (we are on 0.37.6): https://github.com/electron/electron/releases/tag/v0.37.7